src/resource_clients/dataset.ts

```diff
@@ -73,14 +74,20 @@ export class DatasetClient<
             }),
         );
 
-        const response = await this.httpClient.call({
-            url: this._url('items'),
-            method: 'GET',
-            params: this._params(options),
-            timeout: DEFAULT_TIMEOUT_MILLIS,
-        });
+        const listItems = async (
```

Comment:
nit: can we rename this so it doesn't clash with the enclosing method? maybe `fetchItems` ?